index.ts

```diff
@@ -63,14 +70,19 @@ async function main() {
                     // if issue is "closed" then skip it, no need to copy/paste already "closed" issues
                     if (projectIssue.state === 'closed') continue;
                     // create a new issue
+                    const issueDetails = await octokit.rest.issues.get({
```

Comment:
here `projectIssue = issueDetails` so we don't need to call API to get an issue info 1 more time, simply use `projectIssue`

Reply:
fixed